pkg/catalogv2/system/manager_test.go

```diff
@@ -36,3 +50,374 @@ func TestManagerRemovesRelease(t *testing.T) {
 	manager.Remove("cattle-system", "rancher-webhook")
 	assert.Equal(t, map[desiredKey]map[string]any{}, manager.desiredCharts)
 }
+
+func TestStart(t *testing.T) {
+	t.Parallel()
+
+	ctrl := gomock.NewController(t)
+	ctx := context.Background()
+
+	mockSettingsController := fake.NewMockNonNamespacedControllerInterface[*v3.Setting, *v3.SettingList](ctrl)
+	mockSettingsController.EXPECT().OnChange(ctx, "system-feature-chart-refresh", gomock.Any())
+
+	mockClusterRepos := fake.NewMockNonNamespacedControllerInterface[*catalog.ClusterRepo, *catalog.ClusterRepoList](ctrl)
+	mockClusterRepos.EXPECT().OnChange(ctx, "catalog-refresh-trigger", gomock.Any())
+
+	manager := Manager{
+		settings:     mockSettingsController,
+		clusterRepos: mockClusterRepos,
+	}
+
+	manager.Start(ctx)
+}
+
+func TestInstallCharts(t *testing.T) {
```

Comment:
Could this test with its subtests be run in parallel?

Reply:
They indeed can, and do now. It doesn't seem to change anything in terms of execution time on my machine, but it doesn't do any harm. Thanks!